File: Year 1 Semester 1 (Sem 1)/Computer Lab/Programming/Module 3/question2.py
s = int(input())
if s < 0:
    print('Enter Positive Number Only')
else:
    if s <= 10:
        print(s * 15)
    elif s <= 90:
        print(150 + (s - 10) * 8)
    else:
        print(150 + 720 + (s - 100) * 6)

File: Year 1 Semester 2 (Sem 2)/Structured and Object-Oriented Programming/Programming/Module 1/question2.c
// 21BDS0340 Abhinav Dinesh Srivatsa

#include <stdio.h>

int main()
{
    int num;
    scanf("%d", &num);
    if (num > 0)
    {
        if (num % 2 == 0)
        {
            printf("even");
        }
        else
        {
            printf("odd");
        }
    }
    else{
        printf("Enter only positive number");
    }
}